crates/data_models/src/payments/payment_intent.rs

```diff
@@ -430,12 +430,14 @@ pub struct PaymentIntentListParams {
     pub offset: u32,
     pub starting_at: Option<PrimitiveDateTime>,
     pub ending_at: Option<PrimitiveDateTime>,
+    pub amount: Option<AmountFilter>,
```

Comment:
Why are we not using the `AmountFilter` from `api_models`? We have access for `api_models` in both `data_models` and `storage_impl`.